src/public/manageDomain.ts

```diff
@@ -0,0 +1,150 @@
+const envList = helper.getElement('.envList')
+const environmentVariables: EnvironmentItem[] = []
+const fullDomain = window.location.pathname.split('/').pop()
+document.getElementById('full-domain').innerText = fullDomain
+
+class EnvironmentItem {
+  name: string
+  value: string
+  isValid = false
+
+  constructor(name?: string, value?: string) {
```

Comment:
I would clean up a lot of the work being done in the constructor by abstracting out into private methods in the class.